data/transactions/logic/fields.go

```diff
@@ -230,7 +230,7 @@ var TxnTypeNames = []string{
 }
 
 // map TxnTypeName to its enum index, for `txn TypeEnum`
-var txnTypeIndexes map[string]int
```

Comment:
There are several of these maps whose only purpose is to extract values for insertion into a TEAL program.  TEAL programs only operate on uint64s.  So some of these maps were already set up to map to uint64 so they could be indexed and the value used directly, while others were like this, and the value needed to be converted each time.  At the very least, I felt they should all be done the same way, and I went this direction because the only use was TEAL.  (Fundamentally, the only purpose for the map is to provide a name in TEAL, so TEAL is the only intended user, ever.)